utils/visual-tests/helpers/async-for-each-helper.js

```diff
@@ -0,0 +1,7 @@
+const asyncForEach = async (array, callback) => {
```

Comment:
Lets remove `-helper` postfix from file name. The `helpers` folder implies that all file inside are helper files.